pages/common/gradle.md

```diff
@@ -3,19 +3,19 @@
 > Gradle is an open source build automation system.
 > More information: <https://gradle.org>.
 
-- Compile a package:
+- Assemble outputs and run checks:
 
 `gradle build`
 
-- Exclude test task:
+- Exclude the `test` task:
 
 `gradle build -x {{test}}`
 
-- Run in offline mode to prevent gradle from accessing the network during builds:
+- Run in offline mode to prevent Gradle from accessing the network during build:
 
 `gradle build --offline`
 
-- Clear the build directory:
+- Clear the output directory:
 
 `gradle clean`
 
```

Comment:
I don't know what to do with `assembleRelease`, it seems specific to Android but I could not find the official doc about it. :shrug:

Reply:
Do you mean it can only be executed on Android (in that case we should move it to `android/`) or it is only used for Android?

Reply:
No, I mean it may only be useful when you develop Android apps. 🤓

Reply:
But I cannot find any official doc about it (is the command obsolete? 🤷).